http/1.6/modules/ROOT/pages/http-request-configuration.adoc

```diff
@@ -0,0 +1,558 @@
+
+If the external resource does not manage sticky sessions, add the service host name to the `mule.http.disableRoundRobin` system property. The HTTP request won't use round-robin DNS to connect to the configured host.
+
+Configure this system property when starting Mule by adding: +
+`./mule -M-Dmule.http.disableRoundRobin=serverhostname.com`
+
+== Configure Response Validator
+
+When the HTTP *Request* operation receives an HTTP response, the operation validates the response through the response status code. By default, HTTP Connector throws an HTTP error response when the status code is 400 or above. Consequently, if the server returns a 404 (Resource Not Found) or a 500 (Internal Server Error) a failure occurs and the connector triggers the error handling.
+
+Two ASCII `..` full stop characters define the range of failure status codes. Any value from 500 through 599 is considered a failure and raises an error. If the HTTP response has any other status code value, it's considered a success.
+
+The HTTP *Request* operation adds the following attributes to the Mule message when receiving an HTTP response:
+
+* `attributes.statusCode` +
+Code representing the status of the HTTP response
+* `attributes.reasonPhrase` +
+A phrase describing the reasoning behind the HTTP response
+
+For a more detailed example of an HTTP response body during an HTTP request call, refer to the  xref:mule-runtime::mule-error-concept.adoc#http-request-error[HTTP Error Response example] documentation.
+
+In the HTTP *Request* operation configuration, you can customize HTTP responses by configuring the *Response validator* field to any of these options:
+
+* *None* +
+Uses the default validator, which throws an error when the status code is greater than or equal to 400.
+* *Expression or Bean reference* +
+Uses a DataWeave expression or bean reference to define the status code.
+* *Failure status coded validator* +
+Defines failure status codes that throw errors.
+* *Success status code validator* +
+Defines status codes that indicate success.
+
+The following example shows how to configure the *Response validator* field in Studio:
+
+. Select the HTTP *Request* operation from your flow.
+. In *General*, click the *Response* tab.
+. Set *Response validator* to *Success status code validator*.
+. In *Values*, add the list of acceptable status codes, separated by commas, such as `200,201`. +
+ If the HTTP response returned has any other status value, the operation fails and raises an error.
+
+image::http-request-13.png[Response validator field set to Success status code validator ]
+
+In the *Configuration XML* editor, the `response-validator` configuration looks like this:
+
+[source,xml,linenums]
+----
+<http:request method="POST" config-ref="HTTP_Request_configuration" path="testpath">
+    <http:response-validator >
+      <http:success-status-code-validator values="200,201" />
```

Comment:
Continuing the previous comment, here you are enumerating the values 200 and 201. The other syntax allows you to specify a range: "200..209"